docs/how-to/join-a-network.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Joining a network
+sidebarDepth: 0
+description: Learn how to join one of the various Filecoin testnets.
+---
+
+# Joining a network
+
+Once a Lotus node has been initialised on the appropriate [Filecoin network option](https://docs.filecoin.io/how-to/networks/), the instructions within this section will 
+
+### Note: Using the Lotus Node from China
+
+If you are trying to use `lotus` from China, set this **environment variable** on your machine:
+
+```sh
+IPFS_GATEWAY="https://proof-parameters.s3.cn-south-1.jdcloud-oss.com/ipfs/"
+```
+
+## Get started
+
+Start the **daemon** using the default configuration in `./build`:
+
+```sh
+lotus daemon
+```
+
+In another terminal window, check your connection with peers:
+
+```sh
+lotus net peers | wc -l
+```
+
+In order to connect to the network, you need to be connected to at least 1 peer. If you’re seeing 0 peers, read our [troubleshooting instructions](https://docs.filecoin.io/mine/mining-troubleshooting/).
+
+Make sure that you have a reasonable "open files limit" set on your machine, such as 10000. If you're seeing a lower value, such as 256 (default on macOS), read our [troubleshooting instructions](https://docs.filecoin.io/mine/mining-troubleshooting/) on how to update it prior to starting the Lotus daemon.
+
+## Chain sync
+
+While the daemon is running, the next requirement is to sync the chain. Run the command below to view the chain sync progress. To see current chain height, visit the appropriate network statistics page.
+
+```sh
+lotus sync wait
+```
+
+- This step will take anywhere between a few hours to a couple of days.
+- You will be able to perform Lotus operations after it is finished.
+
+## Create your first address
+
+Initialize a new wallet:
+
+```sh
+lotus wallet new
+```
+
+Sometimes your operating system may limit file name length to under 150 characters. You need to use a file system that supports long filenames.
+
+Here is an example of the response:
+
+```sh
+t1aswwvjsae63tcrniz6x5ykvsuotlgkvlulnqpsi
+```
+
+- Use an appropriate faucet to acquire FIL
```

Comment:
Lines 64-67 need updating to reflect that you now must use the CLI. See issue description for info.